src/store/removeTool.js

```diff
@@ -0,0 +1,32 @@
+import { state } from './index.js';
+
+/**
+ * Removes all tools from the target element with the provided name
+ *
+ * @export
+ * @param {*} element
+ * @param {baseTool} tool
+ */
+const removeToolForElement = function (element, toolName) {
+  const toolIndex = state.tools.findIndex(
+    (tool) => tool.element === element && tool.name === toolName
+  );
+
+  if (toolIndex >= 0) {
+    state.tools.splice(toolIndex, 1);
+  }
+};
+
+/**
+ * Removes all tools from all enabled elements with the provided name
+ *
+ * @export
```

Comment:
This is a nice tag, we should generally try to flesh out the jsdocs a little more at some point. A tangent, but I also like @api on anything that's meant to be an endpoint, would allow us to auto-generate api docs. I digress.

Reply:
Agreed. I want to finish up the normal docs first, but then circle back and identify best practices for `jsdocs` and how they influence our autogenerated API docs.